api/src/data_providers/clients/mongodb/MongoDatabaseClient.py

```diff
@@ -24,8 +24,8 @@ def wipe_db(self):
         for database_name in databases_to_delete:
             self.database.client.drop_database(database_name)
 
-    def delete_collection(self, collection: str):
-        self.database[collection].drop()
+    def delete_collection(self):
+        self.collection.drop()
```

Comment:
Let's face it - this class implementation only supports one collection. So we don't need to use it as an argument.